src/com/dojo/foundations/parking/cell/ParkingLot.java

```diff
@@ -13,4 +13,12 @@ public ParkingLot(List<Cell> parkingLot) {
     public List<Cell> getParkingLot() {
         return parkingLot;
     }
+
+    public long getParkingLotOccupation() {
+        return this.getParkingLot().stream().filter(entry -> !entry.isFree()).count();
+    }
+
+    public long getFreeCells() {
+        return this.getParkingLot().stream().filter(Cell::isFree).count();
```

Comment:
Same.  Very good here, but can you write an implementation that let me get the free cells by vehicle type too?